apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -147,6 +155,7 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
                 onClick={() => setSelectedTestIndex(index)}
                 style={fluentTesterStyles.testListItem}
                 testID={description.testPage}
+                componentRef={index === 0 ? ref : undefined}
```

Comment:
If you prefer avoiding the word `undefined` :

` ...(index === 0 && {componentRef: ref})`

Reply:
Thanks for the syntax suggestion!